README.md

```diff
@@ -17,9 +17,9 @@ Currently, the following analyzers are included:
 - PNG
 - Portable Executable (PE) <sup>[2]</sup>
 - [STUN](https://github.com/zeek/spicy-analyzers/tree/main/analyzer/stun)
-- [Tailscale VPN](https://github.com/zeek/spicy-analyzers/tree/main/analyzer/tailscale)
+- Tailscale deprecated by <https://github.com/corelight/zeek-spicy-wireguard>
```

Comment:
Let's instead pull this in as a dependency, see comments in https://github.com/zeek/spicy-analyzers/pull/102#pullrequestreview-815950402.